src/ActionLog.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Log from './Action';
+
+const ActionLog = props => {
+  // Setup is similar to ContactList.js
+  return (
+    <ul className="Action-Log">
+      {props.actionLog.map(log => {
```

Comment:
`.map` actually receives two arguments, along with the element in the array it receives the `index` of that element.

So `props.actionLog.indexOf(log)` is not necessary.

The following could be used and you can pass the second argument, `index` as your `key` prop!

`props.actionLog.map( ( log, index ) => {`